rules/streamline-development-with-npm-and-task-runners/rule.md

```diff
@@ -1,7 +1,7 @@
 ---
 seoDescription: Streamline your .NET development process with NPM and task runners, eliminating tedious tasks and improving collaboration.
 type: rule
-archivedreason:
+archivedreason: "Moved to [Do you know the best package manager for React?](rules/the-best-package-manager-for-react/)"
```

Comment:
Hey @JackDevAU 

I think you don't need "/rules" in the link here...

Reply:
I believe for archived rules it needs the `/rules` but for everything else its just a /rule-uri

Reply:
So after reviewing it - we actually don't need `/rules` woot